spark/sql-13/src/main/scala/org/elasticsearch/spark/sql/EsSparkSQL.scala

```diff
@@ -34,6 +34,7 @@ import org.elasticsearch.hadoop.util.ObjectUtils
 import org.elasticsearch.spark.cfg.SparkSettingsManager
 import org.elasticsearch.hadoop.rest.InitializationUtils
 
+@deprecated(message="Support for Spark 1 is deprecated. Use Spark 2 or 3")
```

Comment:
Two nits: I'd add Apache on first mention and add a period to the last sentence.
```suggestion
@deprecated(message="Support for Apache Spark 1 is deprecated. Use Spark 2 or 3.")
```